tempesta_fw/t/unit/test_http_sticky.c

```diff
@@ -151,7 +151,7 @@ static struct {
 	char		*rmark_val;
 	TfwHttpReq	*req;
 	TfwHttpResp	*resp;
-	TfwConn		conn_req;
+	TfwCliConn	conn_req;
 	TfwConn		conn_resp;
```

Comment:
Maybe to replace `TfwConn` here with `TfwSrvConn`, as it was done with `TfwCliConn`?

Reply:
Why? Now we use only `TfwConn` and we are not going to extend the test since it's deprecated, see https://github.com/tempesta-tech/tempesta-test/issues/74